src/pages/index.vue

```diff
@@ -22,10 +22,21 @@
       </h2>
       <VSearchBar
         v-model.trim="searchTerm"
-        class="max-w-[40rem] mt-4 lg:mt-8"
-        :placeholder="featuredSearchText"
+        class="max-w-[40rem] mt-4 lg:mt-8 relative"
+        :placeholder="featuredSearch.term"
         @submit="handleSearch"
+        @blur="toggleAnimatedText"
+        @focus="toggleAnimatedText"
       >
+        <span
+          v-show="showAnimatedText && searchTerm === ''"
+          type="text"
+          aria-hidden
+          class="bg-white ml-4 w-[30ch] rounded-sm absolute block left-0 pointer-events-none text-dark-charcoal-60 group-hover:bg-dark-charcoal-06 group-focus:bg-dark-charcoal-06 group-focus-within:bg-dark-charcoal-06"
+          @click="toggleAnimatedText"
```

Comment:
Since it has pointer-events set to `none`, will this `@click` be fired at all?

Reply:
No, and it's not needed. Thanks for noticing